modules/cluster-logging-json-log-forwarding.adoc

```diff
@@ -0,0 +1,48 @@
+[id="cluster-logging-json-log-forwarding_{context}"]
+= JSON log forwarding
+
+Forwarding JSON logs gives fluentd, Elasticsearch, and other JSON-aware log collectors direct access to JSON log entries as JSON objects.
+
+To illustrate how this works, suppose that you have the following structured JSON log entry.
+
+.Example structured JSON log entry
+[source,yaml]
+----
+{"level":"info","name":"fred","home":"bedrock"}
+----
+
+Ordinarily, the `ClusterLogForwarder` custom resource (CR) forwards that log entry in the `message` field. The `message` field contains the JSON-quoted string equivalent of the JSON log entry, as shown in the following example.
```

Comment:
Ordinarily - can it be considered to be a somewhat advanced vocabulary example? The one that might send some non-native speakers to the dictionaries? "Usually", "normally" or even "often" look less intimidating.

Reply:
Updated. Thanks.